Scripts/sims4communitylib/utils/common_omega_utils.py

```diff
@@ -0,0 +1,32 @@
+from distributor.ops import Op
+from protocolbuffers.Consts_pb2 import MGR_UNMANAGED, MSG_OBJECTS_VIEW_UPDATE
+from protocolbuffers.DistributorOps_pb2 import Operation
+from protocolbuffers.Distributor_pb2 import ViewUpdate
+from server.client import Client
+from sims4communitylib.utils.misc.common_game_client_utils import CommonGameClientUtils
+
+
+class CommonOmegaUtils:
+    """ Utilities for sending messages to the client via the Omega channel. (Improved S4MP compatibility)
+
+    Messages sent through this channel don't replicate to the rest of the clients in a multiplayer S4MP session.
+    """
+
+    @classmethod
+    def send_view_update_message(cls, op: Op, manager_id=MGR_UNMANAGED, object_id=0, client: Client = None):
```

Comment:
I would rename this function to apply Operations rather than simply "view updates". Since it could theoretically be used to apply any type of operations.

Reply:
I think it'd be good to return a `CommonExecutionResult.TRUE` if this is successful, and `CommonExecutionResult.FALSE` if not successful.

Reply:
I called it "view_update" because that is what the Omega message type is called, there are others such as MSG_SELECTABLE_SIMS_UPDATE, MSG_PIE_MENU_CREATE, so on at EA/generated/protocolbuffers/Consts_pb2.py:64